### ISSUE TITLE
background_load failed to load

### ISSUE THREAD

**Issue by radubolovan** [godotengine/godot-demo-projects#266]
Godot version: 3.0.5 official stable

I just downloaded the "godot-demo-projects" zip and the editor doesn't want to load the "misc/background_load" project.

Here's the console's error:
```
ERROR: _load_settings_text: Condition ' config_version > 3 ' is true. returned: ERR_FILE_CANT_OPEN
   At: core/project_settings.cpp:495
```

I managed to load the project by manually editing the "project.godot" file and replace the
`config_version=4`
with
`config_version=3`

However, you might want to fix the problem in the official release of the demos.
Cheers,
Radu

**Comment by aaronfranke** [comment]
Godot 3.1 has been released, this is no longer an issue if you upgrade to Godot 3.1.

**Comment by akien-mga** [comment]
Fixed by #333.
